.github/CHANGELOG.md

```diff
@@ -84,6 +84,16 @@
 
 <h3>Breaking changes</h3>
 
+- The `QNode.metric_tensor` method has been deprecated, and will be removed in an upcoming release.
+  Please use the `qml.metric_tensor` transform instead.
+  [(#1638)](https://github.com/PennyLaneAI/pennylane/pull/1638)
+
+- The utility function `qml.math.requires_grad` now returns `True` when using Autograd
```

Comment:
This is not going in 0.18 right? I'm just thinking now whether we will have to update some of the demos / other docs to incorporate this.

Reply:
Nope, this is being merged into master only/v0.19-dev